projects/js-packages/charts/src/components/conversion-funnel-chart/conversion-funnel-chart.tsx

```diff
@@ -401,22 +338,19 @@ export const ConversionFunnelChart: FC< ConversionFunnelChartProps > = ( {
 
 								{ /* Funnel Bar */ }
 								<div
-									className={ clsx(
-										styles[ 'bar-container' ],
-										isClicked && styles.selected,
-										isBlurred && styles.disabled
-									) }
+									className={ clsx( styles[ 'bar-container' ], isBlurred && styles.disabled ) }
 									onClick={ stepHandlers.get( step.id )?.onClick }
 									onKeyDown={ stepHandlers.get( step.id )?.onKeyDown }
 									role="button"
 									tabIndex={ isBlurred ? -1 : 0 }
 									aria-label={ step.label }
+									style={ { backgroundColor: barBackgroundColor } }
 								>
 									<div
-										className={ clsx( styles[ 'funnel-bar' ], isClicked && styles.selected ) }
+										className={ styles[ 'funnel-bar' ] }
 										style={ {
 											height: `${ barHeight }%`,
-											backgroundColor: primaryColor,
+											backgroundColor: barColor,
```

Comment:
This inline style is overriding any CSS variable set, so removing support will not effect any existing usage.